### PR TITLE
[CPU] fix NormalizeL2 + Multiply

### PR DESCRIPTION
**Pull request by mandrono**
Issue: 32526

### CODE REVIEW

**Comment by mandrono** [comment]
@maxnick @nshchego @a-sidorova
could you review please?

**Review by a-sidorova — APPROVED** [review]

**Comment by mandrono** [comment]
@a-sidorova thanks!
@dmitry-gorokhov could you take a look please?

File: inference-engine/tests/functional/plugin/cpu/single_layer_tests/normalize.cpp [hunk 1]
```diff
@@ -1,70 +1,67 @@
-// Copyright (C) 2020 Intel Corporation
+// Copyright (C) 2021 Intel Corporation
```
Comment (dmitry-gorokhov) [inline inference-engine/tests/functional/plugin/cpu/single_layer_tests/normalize.cpp:1]: 2020-2021

File: inference-engine/tests/functional/plugin/cpu/single_layer_tests/normalize.cpp [hunk 2]
```diff
@@ -1,70 +1,67 @@
...
-        }
+        function = makeNgraphFunction(netPrc, params, normalize, "Normalize");
 
+        selectedType = "unknown_" + std::string(netPrecision.name());
         threshold = 0.015f;
-
-        normalize_l2->get_rt_info() = getCPUInfo();
-
-        function = std::make_shared<ngraph::Function>(results, param, "Normalize");
+        posDepends = false;
```
Comment (dmitry-gorokhov) [inline inference-engine/tests/functional/plugin/cpu/single_layer_tests/normalize.cpp:64]: We for sure need more meaningful name: smt like `checkFusingPosition`. BTW why we shouldn't do it for this test?

Comment (mandrono) [reply inference-engine/tests/functional/plugin/cpu/single_layer_tests/normalize.cpp]: ```ConvertNormalizeL2WithMulToNormalizeIE``` is ngraph transformation. It creates fusing names used ```FusedNames``` attributes, which is not depends from fused node position.

**Review by dmitry-gorokhov — APPROVED** [review]

**Pull request merged**